dapps/src/router.rs

```diff
@@ -109,6 +112,7 @@ impl http::RequestMiddleware for Router {
 				=>
 			{
 				trace!(target: "dapps", "Resolving to 404.");
+				refresh.unwrap().refresh_local_dapps();
```

Comment:
I didn't want to dispute this, but now that I am re-incorporating it, I want to do it as so SINCE the code is only run upon the condition `self.endpoints.is_some()`